python/examples/tabulated.py

```diff
@@ -0,0 +1,156 @@
+from scipy.special import spherical_jn as j_l
+
+from rascaline import SphericalExpansion, generate_splines
+
+
+########################################################
+# Spherical Bessel zeros (from the scipy cookbook)
+
+
+def Jn(r, n):
+    return sqrt(pi / (2 * r)) * jv(n + 0.5, r)
+
+
+def Jn_zeros(n, nt):
+    zerosj = zeros((n + 1, nt), dtype=np.float64)
+    zerosj[0] = arange(1, nt + 1) * pi
+    points = arange(1, nt + n + 1) * pi
+    racines = zeros(nt + n, dtype=np.float64)
+    for i in range(1, n + 1):
+        for j in range(nt + n - i):
+            foo = brentq(Jn, points[j], points[j + 1], (i,))
+            racines[j] = foo
+        points = racines
+        zerosj[i][:nt] = racines[:nt]
+    return zerosj
+
+
+########################################################
+
+
+# Hyperparameters:
+lmax = 10
+nmax = 10
+a = 5.0  # Radius of the LE sphere - i.e, the cutoff radius
+
+# Create an array with the spherical Bessel zeros:
+z_ln = Jn_zeros(lmax, nmax)
+z_nl = z_ln.T
+
+
+def R_nl(n, el, r):
+    # Un-normalized LE radial basis functions
+    return j_l(el, z_nl[n, el] * r / a)
+
+
+def N_nl(n, el):
+    # Normalization factor for LE basis functions, excluding the a**(-1.5) factor
+    def function_to_integrate_to_get_normalization_factor(x):
+        return j_l(el, x) ** 2 * x**2
+
+    integral, _ = sp.integrate.quadrature(
+        function_to_integrate_to_get_normalization_factor, 0.0, z_nl[n, el]
+    )
+    return (1.0 / z_nl[n, el] ** 3 * integral) ** (-0.5)
+
+
+def get_LE_function(n, el, r):
+    R = np.zeros_like(r)
+    for i in range(r.shape[0]):
+        R[i] = R_nl(n, el, r[i])
+    return N_nl(n, el) * R * a ** (-1.5)
+
+
+# Quick normalization check:
+def random_LE_function(x):
+    return get_LE_function(1, 1, x) ** 2 * x**2
+
+
+normalization_check_integral, _ = sp.integrate.quadrature(random_LE_function, 0.0, a)
+print(f"Normalization check (needs to be close to 1): {normalization_check_integral}")
+
+
+# Now the derivatives (by finite differences):
+def get_LE_function_derivative(n, el, r):
+    delta = 1e-6
+    all_derivatives_except_at_zero = (
+        get_LE_function(n, el, r[1:] + delta) - get_LE_function(n, el, r[1:] - delta)
+    ) / (2.0 * delta)
+    derivative_at_zero = (
+        get_LE_function(n, el, np.array([delta / 10.0]))
+        - get_LE_function(n, el, np.array([0.0]))
+    ) / (delta / 10.0)
+    return np.concatenate([derivative_at_zero, all_derivatives_except_at_zero])
+
+
+# The radial basis functions and their derivatives can be input into a spline
+# generator. This will output the positions of the spline points, the
+# values of the basis functions evaluated at the spline points, and the
+# corresponding derivatives.
+positions, values, derivatives = generate_splines(
+    get_LE_function, get_LE_function_derivative, nmax, lmax, a
+)
+
+# Feed the splines to the Rust calculator:
+# (IMPORTANT: "atomic_gaussian_width" and "cutoff_function" will be ignored)
```

Comment:
`atomic_gaussian_width` I see why, but this is not the case for `cutoff_function` (or `radial_scaling`), since there are added after the radial integral evaluation

Reply:
Really? I thought I had tried changing them at some points and they seemed to have no effect. I will try again

Reply:
You're right. I was hallucinating: both radial scaling and the cutoff function are not ignored